testplans/bitswap/_compositions/local.toml

```diff
@@ -0,0 +1,30 @@
+[metadata]
```

Comment:
Same as above, I suggest we rename this composition to `small-docker.toml` so that it is clear that it runs on the `local:docker` runner.